src/notifications/notification.jade

```diff
@@ -0,0 +1,5 @@
+span.slds-assistive-text {{assistiveText || severity || 'Info'}}
+button.slds-notify__close(*ngIf="showClose", type="button", nglButtonIcon="inverse", (click)="close($event)")
+  ngl-icon-button(icon="close")
+  span.slds-assistive-text {{closeAssistiveText || 'Close'}}
```

Comment:
Do you believe it would be better to move the default values in the component instead of template?